doma-spring-boot-samples/doma-spring-boot-sample-docker-compose/src/test/resources/application.properties

```diff
@@ -1,13 +1,12 @@
 # Enable Docker Compose in tests
 spring.docker.compose.skip.in-tests=false
```

Comment:
Remove src/test/resources/application.properties. Using src/main/resources/application.properties should be sufficient